docs/bbe-guidelines/bbe-guidelines.md

```diff
@@ -0,0 +1,122 @@
+# Guidelines for Writing Ballerina Examples 
+
+- [General best practices](#general-best-practices)
+- [Practices related to the code comments](#practices-related-to-the-code-comments)
+- [Practices related to the folder structure](#practices-related-to-the-folder-structure)
+
+## General best practices
+
+1. Format the Ballerina source using an IDE Plugin.
+
+2. Always, think about the length of the example. A good example should be short.
+
+3. Make the examples self-contained (i.e., avoid depending on concepts other than the one being illustrated) as much as possible.
+
+4. As a practice, use `ballerina/io` methods in main examples and `ballerina/log` methods in services examples. E.g., Do not use both `io:println` and `log:printInfo` in the same example.
+
+5. It is recommended to limit the maximum character count per line to 80 in `.bal` files. Otherwise, the lines get automatically wrapped in the [Ballerina By Example](https://ballerina.io/learn/by-example/) reducing the readability as shown below.
+
+    Example;
+
+    <img src="images/line-wrap.png" width="600" height="70" alt="wrapping lines" />
+
+6. All keywords and any other word, which needs to be highlighted should be added within backquotes (e.g., `xml`). Do not use a single quote as it will not get highlighted. 
+
+    Example;
+
+    - In the code:
+
+        <img src="images/backquoted-keywords.png" width="400" height="58" alt="backquoted keywords" />
+
+    - In the Baleerina website:
+
+        <img src="images/keyword-in-bio.png" width="400" height="80" alt="keywords in bio" />
+
+    > **Note:** However, the context should be taken into account when formatting words/phrases with backquotes. For example, although the word "record" is a keyword of the language, it need not be highlighted if you are referring to the record construct of the language as shown below.
+
+    ````
+    // Defines an open record type named Student. The { and } delimiters indicate that in addition to the defined fields,
+    // this record type allows additional fields with pure-typed (i.e., anydata|error) values.
+    // The descriptor record { } is equivalent to record {| (anydata|error)...; |}.
+    type Student record {
+        string name;
+        int age;
+    Grades grades;
+    };
+    ```
```

Comment:
Seems the above content is not properly visible in the page. End ``` is also visible and horizontal scroll bar is added.

Reply:
Fixed.